addons/im_livechat/controllers/main.py

```diff
@@ -173,6 +173,22 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                      "scrollUnread": False,
                  },
              )
+        create = []
+        for member in channel.channel_member_ids:
+            if not member.channel_member_history_ids:
```

Comment:
How could the history already exist while the channel/member was just created?
Can't we only rely on `add_members` to do this? It looks like a duplicate of `create_for_member` and this method is already too long/complex/handling too much things?